diagnostic.md

```diff
@@ -47,12 +47,12 @@ Record your responses inside the fenced code blocks below each question.
     If we navigate in the browser to `/movies/123`, how can we reference the
     value `'123'` inside a Route?
 
-    ```md
-    <!-- your response here -->
+    ```
+I'm not sure I understand what you're asking.  We could hard code it in - '/movies/:123'.  But :movie_id is already referncing that value, ian't it?  Just not specifically.  Or are you asking how we would reference it inside the template? With {{movie_id}}?
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
-    ```md
-    <!-- your response here -->
+    ```
+In an number of ways.  We can iterate through using #each and specify what we want the individual elements to be named (i.e. '|item|'), or we can simply call out the labels of the data that is being passed (i.e. 'title' 'name').  Or we can call on a component(which then calls on the data).  In any instance we would use handlebars to do this.
     ```
```

Comment:
I would have counted this if you had just used the word model once. I know you understand that though so 1/2 for this. Don't overthink the answer in this case; we reference data provided by a route with the model.